portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -251,27 +253,69 @@ function allowUserSubmit() {
         }
       });
 }
-google.charts.load('current', {'packages': ['corechart']});
-google.charts.setOnLoadCallback(drawStateIncarcerationChart);
-
-/** Creates chart and adds it to the page. */
-function drawStateIncarcerationChart() {
-  const stateData = new google.visualization.DataTable();
-  stateData.addColumn('string', 'Race');
-  stateData.addColumn('number', 'Count');
-  stateData.addRows([
-    ['Whites', 201],
-    ['Blacks', 1767],
-    ['Hispanics', 385],
-  ]);
-
-  const stateChartDimensions = {
-    'title': 'Incarceration Rates (per 100,000) by Race in California',
-    'width': 500,
-    'height': 400,
+
+drawTimeSeriesChart();
+
+/** Adds a line chart to page showing the global avg temp from a csv */
+async function drawTimeSeriesChart() {
+  const tempData = await getTempData();
+  const chart = {
+    type: 'line',
+    data: {
+      labels: tempData.xAxis,
+      datasets: [{
+        label: 'Mean Temp',
+        backgroundColor: 'rgb(255, 0, 0)',
```

Comment:
If this ends up being more complex and needing multiple colors, it'd be a good idea to move them to well-named constants.

RED = rgb(255, 0, 0);
GREEN = rgb(0, 255, 0);

Or if intention is more important then:
FAILURE_COLOR = rgb(255, 0, 0);
SUCCESS_COLOR = rgb(0, 255, 0);

Reply:
Will keep this in mind, thanks!